packages/components/src/button/style.scss

```diff
@@ -308,7 +308,8 @@
 	}
 
 	// Toggled style.
-	&.is-pressed {
+	&[aria-pressed="true"],
+	&[aria-pressed="mixed"] {
```

Comment:
After seeing #55004, now I think about it, we could keep using the `is-pressed` class to assign those styles. Afterall, now the `is-pressed` class is assigned when `truthyAriaPressedValues.includes( ariaPressed )`